conference_101719.tex

```diff
@@ -0,0 +1,364 @@
+By setting MAC address of a user device to that of LAP which is on the same path, ARP reply packets from a gateway does not sent to the user device as in usual.
+As aresult, the proposed scheme achieves an accuracy of 96.5\%.
+True positive rate and false positive rate are 31.0\% higher and 9.0\% lower than the previous scheme even in unstable traffic environment. 
+Furthermore, by extending the observation time of ARP procedure, we realize an accuracy of 100.0\% without any error.
+The proposed scheme can detect RAPs accurately even in a real environment where the prevous scheme cannnot.
+\end{abstract}
+
+\begin{IEEEkeywords}
+component, formatting, style, styling, insert
+\end{IEEEkeywords}
+
+\section{INTRODUCTION}
+With the rapid development of wireless communication techniques, Wi-Fi as the most commonly used Internet access technology is deployed and keeps spreading all over the world to everywhere of our daily life, such as shopping malls, restaurants, public transit systems and so on.
+While the free access to Wi-Fi network attarcts a large number of users, it also allows an adversary to launch attacks to users just by setting up a rogue AP (RAP) in the network.
+An Evil Twin Attack is one of the attack where a RAP is established between a user device and the Internet as man-in-the-middle.
+The RAP clones the service set identifier (SSID) and or even MAC address of a legitimate AP provided by a public facility, which is the reason why it is called "evil twin attack".
+By acting as man-in-the-middle and observing relayed packets between a user device and LAP, an adversary can eavesdrop on the exchange of sensitive information such as identity credentials, passwords, and bank account.
+In addition, an adversary can also mount an acctive attack by leading a user to phishing websites or infecting a user device with malicious softwares.
+Since, our wireless devices connect automatically to an AP which has the strongest received signal strength indication (RSSI) of all APs which have same SSID and moreover they are provided Internet services as usual, an adversary can succed an attack without being noticed by a user.
+Thus, the detection of the RAPs is urgent demand.
+
+The attack is divided into two models based on how a RAP provide the Internet service to user device.
+A model where a RAP use mobile communication can be detected easily with Internet Services Provider (ISP) names or Global IP addresses\cite{rtt}.
+We focus on another model where a RAP uses same gateway with LAPs in the network by connecting with one of it since it cannot be detected by existing schemes.
+During an early stage, the RAP detection is network administrator side detection based on whitelist-based mechanisms\cite{prapd}\cite{clockskew }.
+However, those solutions are inapplicable to Wi-Fi hotspots since administrators have little motivation to guarantee no attacks by setting up additional devices in their infrastructures.
```

Comment:
```suggestion
However, those solutions are inapplicable to Wi-Fi hotspots since administrators have little motivation to guarantee no attacks by setting up additional devices in their infrastructures.
```
ここはネットワーク管理者サイドがダメな理由としてそのまま使えそう．